flathub.json

```diff
@@ -1,3 +1,3 @@
 {
```

Comment:
you can remove the whole file. no other arch is supported.

Reply:
Done.